tests/test_app_client.py

```diff
@@ -36,6 +37,8 @@
 IDS = [ID]
 NAME = "name"
 CID = "cid"
+PAGE_TOKEN = ""
```

Comment:
Can we put some other value here? The default `string` value is an empty string, so testing against that isn't necessarily going to tell us much.